code/model/modeloctant.cpp

```diff
@@ -19,7 +19,7 @@
 #include "model/model.h"
 #include "model/modelsinc.h"
 
-extern void model_allocate_interp_data(int n_verts, int n_norms, int n_list_verts = 0);
+extern void model_allocate_interp_data(int n_verts, int n_norms);
```

Comment:
This is not the same declaration as in modelread.cpp which has default parameters. In any case, if a function is required more than once then it should be put into a header file instead of using extern declaration.